ImprovedUI/Public/Game/GUI/Widgets/CharacterCreation.xaml

```diff
@@ -692,13 +720,13 @@
                         <ContentControl DataContext="{Binding AppearanceMakeup.Materials}" HorizontalAlignment="Center" Margin="0,20,0,0" Template="{DynamicResource Carousel}"/>
                         <StackPanel HorizontalAlignment="Center">
                             <!-- MOD START - Removing tickbox as not required -->

```

Comment:
If you're reverting this change, you should also remove the mod start/end comment for it